### PR TITLE
Add doc for OWSM

### PR DESCRIPTION
**Pull request by pyf98**
This PR adds documentation for [OWSM](https://arxiv.org/abs/2309.13876) (ASRU 2023).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -874,4 +882,10 @@ Also, we can use this tool to provide token-level segmentation information if we
   journal={arXiv preprint arXiv:2211.17196},
   year={2022}
 }
+@article{peng2023reproducing,
+  title={Reproducing Whisper-Style Training Using an Open-Source Toolkit and Publicly Available Data},
+  author={Peng, Yifan and Tian, Jinchuan and Yan, Brian and Berrebbi, Dan and Chang, Xuankai and Li, Xinjian and Shi, Jiatong and Arora, Siddhant and Chen, William and Sharma, Roshan and others},
+  journal={arXiv preprint arXiv:2309.13876},
```
Comment (ftshijt) [inline README.md:888]: We can set it to proceedings of ASRU in the case?

**Review by ftshijt — COMMENTED** [review]
LGTM!

File: egs2/TEMPLATE/s2t1/README.md [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+<sop> I'm going to talk today about energy and climate.<sos><en><transcribe><0.00> And that might seem a bit surprising, because my full-time work at the foundation is mostly about vaccines and seeds, about the things that we need to invent and deliver to help the poorest two billion live better lives.<14.12><15.36> But energy and climate are extremely important to these people; in fact, more important than to anyone else on the planet.<24.26><eos>
+```
+
+During data preparation, three text files are generated:
+- `text` contains the normal target sentence, i.e., the text between `<sos>` and `<eos>`.
+- `text.prev` contains the previous sentence, i.e., the text between `<sop>` and `<sos>`. This might be unavailable at the beginning of a talk. In such cases, a special token `<na>` will be used.
+- `text.ctc` contains the ASR transcript without any special token, which is used for the CTC loss. For ASR utterances, this can be derived from `text`, but for ST utterances, this is in a different language. If the ASR transcription is not available, `<na>` will be used.
+
+
+### 2. Speed perturbation
```
Comment (sw005320) [inline egs2/TEMPLATE/s2t1/README.md:60]: Do we need sp?
Maybe we can remove it even from the template in the future.

**Review by sw005320 — APPROVED** [review]

File: egs2/TEMPLATE/s2t1/README.md (hunk 2 above)
Comment (pyf98) [reply egs2/TEMPLATE/s2t1/README.md:60]: yeah, we may not need it in the future. We didn't use it previously.

File: README.md (hunk 1 above)
Comment (pyf98) [reply README.md:888]: we can wait until the proceedings actually come out maybe

**Pull request merged**